server/data/educationAndWorkPlanClient.test.ts

```diff
@@ -158,4 +160,51 @@ describe('educationAndWorkPlanClient', () => {
       }
     })
   })
+
+  describe('getActionPlans', () => {
+    it('should get Action Plans', async () => {
+      // Given
+      const prisonNumbers = ['A1234BC', 'B5544GD']
+      const systemToken = 'a-system-token'
+
+      const expectedActionPlanSummaryListResponse = aValidActionPlanSummaryListResponse({
+        actionPlanSummaries: [
+          aValidActionPlanSummaryResponse({
+            reference: '6add2455-30f1-4b3e-a23e-1baf2d761e8f',
+            prisonNumber: 'A1234BC',
+          }),
+          aValidActionPlanSummaryResponse({
+            reference: 'b134fb41-426d-4494-bb66-75dafd9dc084',
+            prisonNumber: 'B5544GD',
+          }),
+        ],
+      })
+      educationAndWorkPlanApi.post('/action-plans', { prisonNumbers }).reply(200, expectedActionPlanSummaryListResponse)
+
+      // When
+      const actual = await educationAndWorkPlanClient.getActionPlans(prisonNumbers, systemToken)
+
+      // Then
+      expect(nock.isDone()).toBe(true)
+      expect(actual).toEqual(expectedActionPlanSummaryListResponse)
+    })
+
+    it('should get zero Action Plans given none of the specified prisoners have Action Plans', async () => {
```

Comment:
This test demonstrates that we expect a response with an empty collection, rather than a 404
Functionally its not massively different from the test above, but it does demonstrate the intent and behaviour 👍